lib/rspecq/queue.rb

```diff
@@ -241,6 +242,22 @@ def flaky_jobs
       requeued - @redis.hkeys(key_failures)
     end
 
+    def fail_fast_threshold
+      @fail_fast || -1
+    end
+
+    # Returns true if the number of failed tests, has surpassed the threshold
+    # to render the run unsuccessful. This is also used internally to block any
+    # new jobs being enqueued
+    def suite_failed?
+      return false if @fail_fast.nil?
+
+      failed_examples = @redis.hlen(key_failures)
+      failed_non_examples = @redis.hlen(key_errors)
```

Comment:
Consider naming to something more consistent with the terminology used throughout the codebase, to avoid confusion:

- num_failed_examples
- num_errors

Reply:
0e65496

Reply:
The commit you linked is irrelevant.